HB/common/database.elpi

```diff
@@ -462,3 +473,8 @@ find-max-classes [M|Mixins] [C|Classes] :-
   ].
 find-max-classes [M|_] _ :- coq.error "HB: cannot find a class containing mixin" M.
 
+pred is-subject-lifter i:term, o:int, o:classname.
```

Comment:
This is wrong if the lifter is casted: `(hom : T -> T -> Type)` hides `(let x : T -> T -> Type := hom in x)`
and this code should cross/reduce the let